### PR TITLE
(mostly) iSCSI iBFT fixes

### PR DESCRIPTION
**Pull request by tbzatek**
A couple more other fixes that I found along the way.

### CODE REVIEW

**Comment by tbzatek** [comment]
Jenkins, test this please.

File: src/tests/dbus-tests/test_30_iscsi.py [hunk 1]
```diff
@@ -374,3 +374,56 @@ def test_login_noauth_badauth(self):
         udisks = self.get_object('')
         objects = udisks.GetManagedObjects(dbus_interface='org.freedesktop.DBus.ObjectManager')
         self.assertNotIn(dbus_path, objects.keys())
+
+    def test_ibft(self):
```
Comment (vojtechtrefny) [inline src/tests/dbus-tests/test_30_iscsi.py:378]: I think we should tag this test as `UNSAFE`, just to be sure we won't try to login to some ibft iscsi when running in some random environment.

**Review by vojtechtrefny — APPROVED** [review]
Looks good to me.

File: src/tests/dbus-tests/test_30_iscsi.py (hunk 1 above)
Comment (tbzatek) [reply src/tests/dbus-tests/test_30_iscsi.py:379]: Yeah, good idea. I don't expect these tests to be ever used except of very isolated cases of specific testing environments.

**Pull request merged**